data-serving/data-service/src/common/db-client.ts

```diff
@@ -0,0 +1,20 @@
+import { MongoClient } from 'mongodb';
+
+class DbClient {
+    client: MongoClient;
+
+    constructor() {
+        this.client = new MongoClient('mongodb://localhost:27017', { useUnifiedTopology: true });
+    }
+
+    public async connect() {
```

Comment:
Does `npm run lint` get mad about these not having types?

Reply:
weirdly no but some types aren't a bad idea -- now only the new db() function will return something so added a type here